.travis.yml

```diff
@@ -8,7 +8,7 @@ rvm:
   - "1.9.3"
   - "1.8.7"
   - "2.0.0"
-  - "2.1"
```

Comment:
This change isn't needed. Since travis uses rvm, a version spec of 2.1 will pick up the latest 2.1.x.